### PR TITLE
DOCS-4415 Using USM Metrics in Monitors, SLOs, and Dashboards Guide

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds Using USM Metrics in Monitors, SLOs, and Dashboards guide and accompanying screenshots.

### Motivation

DOCS-4415

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/universal_service_monitoring/guide/using_usm_metrics.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+This guide describes how to search for USM metrics such as `universal.http.*` and use them in your monitors, SLOs, and dashboards.
+
+## USM metrics vs APM metrics
+
+| Metric Name                 | Units   | Type         | Description                                       |
+|-----------------------------|---------|--------------|---------------------------------------------------|
+| universal.http.client       | Seconds | Distribution | Total number of outbound requests.                |
+| universal.http.client.apdex |         | Gauge        | Total number of outbound errors.                  |
+| universal.http.server       | Seconds | Distribution | Total number of outbound requests by HTTP status. |
+| universal.http.server.apdex |         | Gauge        | The Apdex score for each web service.             |
```
Comment (alai97) [inline content/en/universal_service_monitoring/guide/using_usm_metrics.md:34]: Metric descriptions are TBD.

```suggestion
| universal.http.client       | Seconds | Distribution | Total number of outbound requests.                |
| universal.http.client.apdex |         | Gauge        | Total number of outbound errors.                  |
| universal.http.server       | Seconds | Distribution | Total number of outbound requests by HTTP status. |
| universal.http.server.apdex |         | Gauge        | The Apdex score for each web service.             |
```

File: content/en/universal_service_monitoring/guide/using_usm_metrics.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+| trace.universal.http.client.errors              | count:universal.http.client{error:true}              |
+| trace.universal.http.client.hits.by_http_status | count:universal.http.client{*} by http_status_family |
+| pXX:trace.universal.http.client{*}              | pXX:universal.http.client{*}                         |
+| trace.universal.http.client.apdex{*}            | universal.http.client.apdex{*}                       |
+
+The same translations apply for the `universal.http.server` operation that captures inbound traffic. For more information about distribution metrics, see [DDSketch-based Metrics in APM][12].
+
+## Usage
+
+Navigate to [**APM** > **Service Catalog**][5] and click on a service. The **Performance** tab displays service-level graphs on hits, latency, requests, errors, and more. You can also access these metrics when creating a [monitor](#create-a-monitor) or an [SLO](#create-an-slo), or by looking at a [dashboard](#access-a-defined-dashboard) in the [Service Catalog][2].
```
Comment (kayayarai) [inline content/en/universal_service_monitoring/guide/using_usm_metrics.md:54]: Do we need to specify here that they should look at USM-identified services? Something like:

`Navigate to [**APM** > **Service Catalog**][5], filter by Universal Service Monitoring telemetry type, and click on a service. `

**Review by kayayarai — CHANGES_REQUESTED** [review]
Just a tiny edit and a question. Also, still awaiting metrics descriptions

File: content/en/universal_service_monitoring/guide/using_usm_metrics.md (hunk 2 above)
Comment (alai97) [reply content/en/universal_service_monitoring/guide/using_usm_metrics.md:54]:
```suggestion
Navigate to [**APM** > **Service Catalog**][5], filter by a Universal Service Monitoring telemetry type, and click on a service. The **Performance** tab displays service-level graphs on hits, latency, requests, errors, and more. You can also access these metrics when creating a [monitor](#create-a-monitor) or an [SLO](#create-an-slo), or by looking at a [dashboard](#access-a-defined-dashboard) in the [Service Catalog][2].
```

**Review by kayayarai — APPROVED** [review]
Just waiting on updates to metrics/descriptions

File: content/en/universal_service_monitoring/guide/using_usm_metrics.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+| universal.http.client       | Seconds | Distribution | Outbound request latency, counts, errors, and rates.                |
+| universal.http.client.hits  | Hits    | Count        | Total number of outbound requests and errors.                |
+| universal.http.client.apdex | Score   | Gauge        | The Apdex score of outbound requests for this service.                |
+| universal.http.server       | Seconds | Distribution | Inbound request latency, counts, errors, and rates.  |
+| universal.http.server.hits  | Hits    | Count        | Total number of inbound requests and errors.                 |
+| universal.http.server.apdex | Score   | Gauge        | The Apdex score for this web service.             |
+
+Unlike APM metrics, errors are available under the `error:true` tag instead of as a separate metric.
+
+*Note: The `.hits` metrics have all of your infrastructure tags and are the recommended way to query request and error counts. You can also add [second primary tags][16] to all USM metrics.*
```
Comment (alai97) [inline content/en/universal_service_monitoring/guide/using_usm_metrics.md:40]:
```suggestion
**Note:** The `.hits` metrics have all of your infrastructure tags and are the recommended way to query request and error counts. You can also add [second primary tags][16] to all USM metrics.*
```

**Review by pinkerton — APPROVED** [review]

**Pull request merged**